packages/namekit-react/src/components/PriceSymbol.tsx

```diff
@@ -0,0 +1,88 @@
+import { PriceCurrencyFormat, Currency } from "@namehash/ens-utils";
+import { Tooltip } from "@namehash/nameguard-react";
```

Comment:
What do you think about moving `Tooltip` into NameKit react? Overall I'd prefer that NameGuard react only includes things that are truly nameguard-only.

Reply:
I think it is a good idea!